docs/xgc.md

```diff
@@ -60,6 +62,26 @@ Options that change the behavior of the Garbage Collector (GC).
 
 : This option sets the scan mode for GC operations that evacuate objects in the heap (scavenge operations (`gencon`) and copy forward operations (`balanced`)) to breadth first mode. The scan mode reflects the method for traversing the object graph and is also known as *Cheney's algorithm*.
 
+### `classUnloadingKickoffThreshold`
+
+         -Xgc:classUnloadingKickoffThreshold=<value>
+
+: Where `<value>` is equal to the number of class loaders plus the number of anonymous classes that are loaded since the previous class unloading operation.
```

Comment:
apology for the late comment... but i'm wondering if the better word would be  'Where `<value>` is _compared_ to the number of...'